backend/routes/quests.go

```diff
@@ -80,6 +80,13 @@ type QuestStatus struct {
 	Needed   int `json:"needed"`
 }
 
+type QuestProgress struct {
+	QuestId   int `json:"questId"`
+	QuestType int `json:"questType"`
```

Comment:
Please remove the QuestType field from quest progress, and create a separate struct for `QuestType` used to query the SQL tables.

type QuestType struct {
	QuestId   int `json:"questId"`
	QuestType int `json:"questType"`
	}